tla/consensus/ccfraft.tla

```diff
@@ -746,8 +748,8 @@ BecomeLeader(i) ==
 ClientRequest(i) ==
     \* Only leaders receive client requests (and therefore they have not yet completed retirement)
     /\ leadershipState[i] = Leader
-    \* ... and the leader should not have its retirement signed
-    /\ membershipState[i] # RetirementSigned
+    \* See raft.h::can_replicate_unsafe():911
```

Comment:
Line numbers are brittle.

Reply:
Agreed, removed, although function names are only quantitatively more stable. In a perfect world, Sphinx would support TLA, and this would be `:cpp:func:ccf::aft::Aft:can_replicate_unsafe()` and we would be told as soon as it breaks. Alas.

Reply:
You could add a random text string comment to `raft.h`, and reference that from the TLA+ spec.